foundry.toml

```diff
@@ -15,15 +15,10 @@ fail_on_revert = true
 [profile.default.fmt]
 wrap_comments = true
 
-
 [profile.build]
 test = "/dev/null"
 script = "/dev/null"
 
-
-[profile.test]
-via_ir = false
```

Comment:
`--via-ir=false` works too?

Reply:
> --via-ir=false works too?

Yes but not with `yarn test:forge` which is the recommended way to run test, as per the README. We could also change the `test:forge` target, but it's easier and consistent to the way we have done things in other repos to just keep this `test` profile

Reply:
let me open an issue to remove the necessity of yarn test:forge

Reply:
Sure we can not recommend using `yarn test:forge` to run tests in the README, but why still removing this test profile to run tests faster?
Still holding my point:
> Actually not that fine with it because the thing with foundry profiles is they are opt-in, not defaults, so I would argue in favor of keeping this profile and just use `forge test` if you don't want to use the `test` profile

Reply:
i keep it like that for the moment, let's nitpick in the issue